templates/tools/dockerfile/clang_update.include

```diff
@@ -1,28 +1,34 @@
 #=================
 # Update clang to a version with improved tsan and fuzzing capabilities
 
-RUN apt-get update && apt-get -y install python cmake && apt-get clean
+RUN apt-get update && apt-get -y install python && apt-get clean
 
-RUN git clone -n -b release_38 http://llvm.org/git/llvm.git && ${'\\'}
-  cd llvm && git checkout ad57503 && cd ..
-RUN git clone -n -b release_38 http://llvm.org/git/clang.git && ${'\\'}
-  cd clang && git checkout ad2c56e && cd ..
-RUN git clone -n -b release_38 http://llvm.org/git/compiler-rt.git && ${'\\'}
-  cd compiler-rt && git checkout 3176922 && cd ..
-RUN git clone -n -b release_38 ${'\\'}
+RUN git clone -n -b release_50 http://llvm.org/git/llvm.git && ${'\\'}
```

Comment:
nit: I am not a fan of building clang from source - it takes long time and brings some unnecessary clutter with it. 

Have we looked for binary distribution of clang?